crates/testing/src/overall_safety_task.rs

```diff
@@ -249,12 +260,9 @@ impl<TYPES: NodeType, I: TestableNodeImplementation<TYPES>, V: Versions> TestTas
             expected_views_to_fail,
         }: OverallSafetyPropertiesDescription<TYPES> = self.properties.clone();
 
-        let num_incomplete_views = self
-            .ctx
-            .round_results
-            .len()
-            .saturating_sub(self.ctx.successful_views.len())
-            .saturating_sub(self.ctx.failed_views.len());
+        let num_incomplete_views = self.ctx.round_results.len()
```

Comment:
i think we want to set this back, also i wonder if i should do a check to see if we are hitting a subtraction overflow and fail the test. this way maybe next time it wont be as hard to debug as we couldnt see error in CI previously